Obsolete 4329

linuxwolf opened the pull request:
Do all the work necessary to truly obsolete 4329, replacing it with this draft.

* [x] Change metadata to mark as "obsoletes 4329" instead of "updates 4329"
* [x] Port all relevant wording from 4329 into this draft
  - [x] Ensure Security Considerations still includes this draft's current additions
* [x] Update Abstract and Introduction to mark 4329 as obsolete instead of updated
* [x] Update IANA instructions to reference this document instead of 4329

MylesBorins commented on javascript-mjs/draft-ietf-dispatch-javascript-mjs-05.txt:
```diff
@@ -50,70 +49,128 @@ Copyright Notice
+   type registry, to foster interoperability by defining underspecified
+   aspects, and to provide general security considerations.
+
+   This document updates the existing media types for the ECMAScript
```
Is this still accurate as we are deprecating not updating?

linuxwolf replied:
Clearly I did not re-read the intro (-:

Yes, this needs to be changed to say it's obsoleting not updating.

MylesBorins commented on javascript-mjs/draft-ietf-dispatch-javascript-mjs-05.txt:
```diff
@@ -4,23 +4,22 @@
 
 DISPATCH                                                       M. Borins
```
Should this author list now also include the authors of the original RFC since we are re-using a bunch of the copy?

linuxwolf replied:
The author's/editor's list reflects who was directly responsible for this work, even if it is obsoleting a previous document.  I did add Bjöern Hörmann to the acknowledgements.

As for the existing list order, how about I reorder the list to be alphabetical.  I've really just pulled information from other places into this one place (-:

MylesBorins replied:
Alphabetical seems fair to me, although it still results in my name first 😅

MylesBorins commented on javascript-mjs/draft-ietf-dispatch-javascript-mjs-05.txt:
```diff
@@ -50,70 +49,128 @@ Copyright Notice
+
+   This document defines equivalent processing requirements for the
+   types text/javascript, text/ecmascript, and application/javascript.
+   Use of and support for the media type application/ecmascript is
```
Should it be mentioned that application/javascript and application/ecmascript are considered obsolete?

linuxwolf replied:
That's a good point.  It is called in other places in the document structure, but it would certainly be good to put that in context here.

linuxwolf commented on javascript-mjs/draft-ietf-dispatch-javascript-mjs-05.txt:
```diff
@@ -50,70 +49,128 @@ Copyright Notice
+   programming languages.  Refer to "Brief History" and "Overview" in
+   [ECMA-262] for background information on these languages.
+
+   Programs written in these programming languages have historically
```
This is a direct copy from the original.  I'll work out some better text.

linuxwolf commented:
@MylesBorins @mathiasbynens I think I've addressed all the comments, and I backed out all the generated-but-committed files for now, to make it clearer what is to reviewed.

* 8aa889f — addressing the latest review comments
* "everything" — changes since the last revision

linuxwolf commented on javascript-mjs/draft-ietf-dispatch-javascript-mjs-05.txt:
```diff
@@ -153,22 +378,102 @@ Internet-Draft       ECMAScript Media Types Updates         October 2019
    form of mutual trust.
 
-   All other security considerations from [RFC4329] still apply.
+   Uncontrolled execution of scripts can be exceedingly dangerous.
```
I went back and forth on this one.  I think it still works either way, but happy to change.

MylesBorins replied:
I'm happy if you are happy
